tests/validation_suite/automation_framework/worker_lookup/worker_lookup_params.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+
+import automation_framework.worker.worker_params as worker
+
+logger = logging.getLogger(__name__)
+
+class WorkerLookUp():
+    def __init__(self):
+        self.id_obj = {"jsonrpc": "2.0", "method": "WorkerLookUp", "id": 1}
+        self.params_obj = {}
+
+    def add_json_values(self, input_json_temp, worker_obj, tamper):
+
+        if "workerType" in input_json_temp["params"].keys() :
+            if input_json_temp["params"]["workerType"] != "" :
+                self.set_worker_type(input_json_temp["params"]["workerType"])
+            else :
+                self.set_worker_type(1)
```

Comment:
Can we use enum for worker_type?

Reply:
updated to use values for worker_type from enum

> Can we use enum for worker_type?

Updated to use worker_type value from enum